backend/app/services/openapi/chat_session.py

```
@@ -294,6 +294,43 @@ def setup_chat_session(
+    # Store user message in long-term memory (fire-and-forget)
+    # Only store if enable_chat_bot=True (wegent_chat_bot tool is enabled)
+    # This runs in background and doesn't block the main flow
+    enable_chat_bot = tool_settings.get("enable_chat_bot", False)
+    if enable_chat_bot:
+        import asyncio
+
+        from app.services.memory import get_memory_manager
+
+        memory_manager = get_memory_manager()
+        if memory_manager.is_enabled:
+            task_crd = Task.model_validate(task.json)
```

Ruff (0.14.11)

Undefined name `Task`

(F821)